mautrix_signal/portal.py

```diff
@@ -724,14 +724,18 @@ async def handle_matrix_leave(self, user: u.User) -> None:
 
     async def handle_matrix_invite(self, invited_by: u.User, user: u.User | p.Puppet) -> None:
         if self.is_direct:
-            raise RejectMatrixInvite("You can't invite additional users to private chats.")
+            await self.main_intent.send_notice(
+                self.mxid, "You can't invite additional users to private chats."
+            )
         else:
             try:
                 await self.signal.update_group(
                     invited_by.username, self.chat_id, add_members=[user.address]
                 )
             except RPCError as e:
-                raise RejectMatrixInvite(e.message) from e
+                await self.main_intent.send_notice(
+                    self.mxid, f"Failed to invite {user.mxid} on Signal: {e}"
+                )
```

Comment:
`RejectMatrixInvite` is the correct error for ghost invites, it's just the non-bridge-ghost `handle_invite` which wasn't catching them